src/pages/gateway/hooks.md

```diff
@@ -230,28 +282,34 @@ module.exports = {
 
 #### Remote composer example
 
-```js
-addEventListener("fetch", (event) => event.respondWith(handleRequest(event)));
+This remote composer fetches your authorization token and inserts it into the `x-auth-token` header.
```

Comment:
This is a local composer as well. Lets use this example about how one can insert a new header into existing request headers.

You can move this to the local composer example section.

Reply:
As an example remote composer, use this which is exactly the same as the above local composer that does auth header checking but on fastly edge compute.

Users can use any serverless function of their choice but for the sake of example here is one that runs on Fastly edge compute.

```js
addEventListener("fetch", (event) => event.respondWith(handleRequest(event)));

async function handleRequest(event) {
  try {
    const body = await event.request.json();
    if (!body.context.headers["authorization"]) {
      return new Response(
        { status: "SUCCESS", message: "Unauthorized" },
        { status: 401 }
      );
    }
    return new Response(
      { status: "SUCCESS", message: "Authorized" },
      { status: 200 }
    );
  } catch (err) {
    return new Response(err, { status: 500 });
  }
}
```